SETUP.md

```diff
@@ -61,6 +61,21 @@ Assuming the repo is cloned as `Recommenders` in the local system, to install **
 
 You can specify the environment name as well with the flag `-n`.
 
+<details>
```

Comment:
what are you thoughts about putting this in the section for PySpark setup? do you think that hides it too much? If so perhaps keeping the note here, but referring to Pyspark Environment setup for details on installing Java dependencies would be more organized.

Reply:
Yeah, I thought about this one for some time. Initially I've placed this under PySpark environment setup. But then I thought that I would need to duplicate this piece and place it under Full (PySpark & Python GPU) environment. Hence I've placed it above those two. If you think having this piece twice under those two section makes more sense I'll move it.

Reply:
I think it would be great to put the note that using PySpark 2.4.x requires Java 8 underneath the note about xlearn so it's visible at the top level. We can add a reference there to PySpark Environment section for more details.

we should add a section in PySpark environment about Installing Java for Mac.

The Full environment section already references PySpark, so no need to repeat it there.

Reply:
Will this work?